lib/internal/fs/promises.js

```diff
@@ -99,6 +100,13 @@ const {
 const getDirectoryEntriesPromise = promisify(getDirents);
 const validateRmOptionsPromise = promisify(validateRmOptions);
 
+let copyPromises;
+function lazyLoadCopyPromises() {
+  if (copyPromises === undefined) {
+    ({ copyPromises } = require('internal/fs/copy/copy'));
+  }
```

Comment:
Wouldn't it make more sense to return the value from the function?
```suggestion
  return copyPromises ??= require('internal/fs/copy/copy').copyPromises;
```